docs/guides/admin/docs/releasenotes.md

```diff
@@ -80,6 +80,10 @@ Configuration changes
 - The role `ROLE_UI_EVENTS_DETAILS_GENERAL_VIEW` for viewing the publications (previously general) tab in the event
   details modal has been renamed to `ROLE_UI_EVENTS_DETAILS_PUBLICATIONS_VIEW` for consistency.
 
+- The introduction of Capture Agent Access Management implies that unprivileged users do not have access to any capture
+  agents anymore. Before Opencast 6.0, unprivileged users had access to all capture agents. In case you are working
```

Comment:
In case you are working... ->
```
To allow otherwise unprivileged users to interact with event scheduling, you need to configure their access permissions appropriately.  The relevant permission is $PERMISSION, which grants allows the user to schedule new events, and edit scheduled events.
```